internal/cli/atlas/livemigrations/link/create.go

```diff
@@ -76,6 +76,7 @@ func CreateBuilder() *cobra.Command {
 	}
 	cmd.Flags().StringVar(&opts.OrgID, flag.OrgID, "", usage.OrgID)
 	cmd.Flags().StringSliceVar(&opts.accessListIP, flag.AccessListIP, []string{}, usage.LinkTokenAccessListCIDREntries)
+	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
```

Comment:
For my own reference, do we normally include this on all commands/why is it needed?

Reply:
we include the output flag for all the commands that return a json response. This flag allows the user to change the output format. Valid values: `json|json-path|go-template|go-template-file`

Reply:
Thanks. I may have to improve some commands :D